pkg/webhook/validate.go

```diff
@@ -48,6 +48,19 @@ func validateSriovOperatorConfig(cr *sriovnetworkv1.SriovOperatorConfig, operati
 	return false, warnings, fmt.Errorf("only default SriovOperatorConfig is used")
 }
 
+func validateSriovNetwork(cr *sriovnetworkv1.SriovNetwork, operation v1.Operation) (bool, []string, error) {
+	glog.V(2).Infof("validateSriovNetwork: %v", cr)
+	var warnings []string
+
+	if operation == v1.Create || operation == v1.Update {
```

Comment:
why we need this? we should only get called by this two operations base on the validationWebhook configuration no?

Reply:
Correct, this improves readability so that people don't have to check the ValidatingWebhookConfiguration object to understand why only update and create are affected. It also prevents future mistakes when other validations are added.

if this bothers you, I could remove it. But I think it's good to have it.